src/main/java/org/terasology/climateConditions/AffectBodyTemperatureEvent.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.climateConditions;
+
+import org.terasology.entitySystem.event.AbstractValueModifiableEvent;
+
+/**
+ * This event is sent out by the {@link BodyTemperatureSystem} to allow for other systems to modify the change in body
+ * temperature.
+ */
+public class AffectBodyTemperatureEvent extends AbstractValueModifiableEvent {
+    private boolean isNegative = false;
+
+    public AffectBodyTemperatureEvent(float baseValue) {
+        super((baseValue > 0) ? baseValue : (-1 * baseValue));
```

Comment:
Ah, I see - the modifiable event does only allow for positive numbers? and this is our way to allow for negative values by making it positive in the underlying event and keeping the state ... 🧐 what about changing `AbstractValueModifiableEvent` instead?

Reply:
I believe you opened this: https://github.com/MovingBlocks/Terasology/pull/4063 for a similar purpose...
And... I actually did leave a comment there.... So... I am on board with changing the AbstarctValueModifiableEvent but we have to be careful in case some other system uses this positive only thing.........